src/patternfly/components/Banner/banner.scss

```diff
@@ -57,4 +57,18 @@
     z-index: var(--pf-c-banner--m-sticky--ZIndex);
     box-shadow: var(--pf-c-banner--m-sticky--BoxShadow);
   }
+
+  .pf-c-button.pf-m-link.pf-m-inline {
+    --pf-c-button--FontWeight: var(--pf-global--FontWeight--semi-bold);
+    --pf-c-button--m-link--Color: var(--pf-c-banner--Color);
```

Comment:
I would create vars for these values instead of reusing the banner text color and using a global var for the font-weight. Then they're customizable by the user via a var without having to write out the `.pf-c-button.pf-m-link.pf-m-inline` selector and redefining the button vars.

Maybe something like

```
// top of scss
--pf-c-banner--link--FontWeight: var(--pf-global--FontWeight--semi-bold);
--pf-c-banner--link--Color: var(--pf-c-banner--Color);
--pf-c-banner--link--hover--Color: var(--pf-c-banner--Color);
--pf-c-banner--link--disabled--Color: var(--pf-c-banner--Color);

// here
--pf-c-button--FontWeight: var(--pf-c-banner--link--FontWeight);
--pf-c-button--m-link--Color: var(--pf-c-banner--link--Color);
...

Reply:
I can remove .pf-m-inline, but  `.pf-c-button.pf-m-link` from the button component will override the variables if I don't create more specificity here.